.github/workflows/build.yml

```diff
@@ -92,11 +92,13 @@ jobs:
     runs-on: ubuntu-22.04
     env:
       NODE: 16.x
+      CHROME_VERSION: ${{ matrix.chrome-version }}
```

Comment:
Not this line, but can we append `${{ matrix.chrome-version }}` to the test name, otherwise CI runs two builds with the same name. Perhaps we could just append it if it's not 90? For example, I don't know if this is value syntax but something like...

```
name: ${{ matrix.cmd }}-${{ matrix.suite || '' }}${matrix.chrome-version === '90' ? '-minimum-browser' : '' }
```